### PR TITLE
Merge profile and user information with oAuth providers.

### PR DESCRIPTION
**Pull request by JAertgeerts**
#106

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,15 @@ module.exports = {
   onUserLogged: (app, user) => {
       return Promise.resolve(user)
   },
+  //Optional: can be used to merge data from the oAuth / oAuth2 profile and the default user properties.
+  mergeThirdPartyProfile: (user, profile) => {
+    console.log(profile)
```
Comment (jaumard) [inline README.md]: not needed I think for the readme

File: README.md [hunk 2]
```diff
@@ -91,6 +91,15 @@ module.exports = {
   onUserLogged: (app, user) => {
       return Promise.resolve(user)
   },
+  //Optional: can be used to merge data from the oAuth / oAuth2 profile and the default user properties.
```
Comment (jaumard) [inline README.md]: It's not only oAuth and oAuth2, it's all third party login. They are mainly these 2 but they're some others

File: README.md [hunk 3]
```diff
@@ -91,6 +91,14 @@ module.exports = {
...
+  mergeThirdPartyProfile: (user, profile) => {
+    var mergedProfile = {
```
Comment (jaumard) [inline README.md]: better to use const as in Trails ES6 is fully embrace (it's a detail ^^)

**Comment by JAertgeerts** [comment]
@jaumard  Can you publish this as a new version?

**Comment by jaumard** [comment]
Do you need it quickly ? Because if @LobeTia (no pressure mate ;)) PR coming in few days with bugs fixes for this functionalities (third party login) it's better to wait for it I think. You can use the master directly until the new version is out. 
Let me know :)

**Comment by JAertgeerts** [comment]
No problem, using master directly in the meanwhile!
Take your time ;)



On Tue, Jan 10, 2017 at 8:24 AM +0100, "Jaumard" <notifications@github.com> wrote:












Do you need it quickly ? Because if @LobeTia (no pressure mate ;)) PR coming in few days with bugs fixes for this functionalities (third party login) it's better to wait for it I think. You can use the master directly until the new version is out.

Let me know :)

**Pull request merged**